common/src/main/resources/assets/lang/it_IT.conf

```diff
@@ -0,0 +1,855 @@
+        economy-block-purchase-limit="&cIl nuovo totale di &a{total}&c eccederebbe dal tuo limite di blocchi di protezione (&a{limit}&c). La transazione è stata annullata."
+        economy-block-sale-confirmation="&aDepositati &6{deposit}&a nel tuo account. Adesso hai &6{amount}&a blocchi di protezione disponibili."
+        economy-block-sell-error="&cImpossibile vendere i blocchi di protezione. Motivo: &f{reason}&c."
+        economy-block-transfer-cancel="&cTrasferimento annullato. Impossibile trovare giocatori con blocchi di protezione."
+        economy-block-transfer-success="&aTrasferimento completato. Hai trasferito correttamente i blocchi di &6{count}&a giocatori nella valuta corrente."
+        economy-block-transfer-warning="&6Sei sicuro di voler trasferire &cTUTTI&6 i blocchi di protezione dei giocatori nella valuta corrente? Questo imposterà i blocchi dei giocatori a 0 dopo il trasferimento. \n\nClicca per confermare."
+        economy-claim-abandon-success="&aZona/e abbandonata/e. Ti sono stati restituiti '&6{amount}&a'."
+        economy-claim-abandon-success-world="&aClaim(s) abandoned in &6{world}&a. You have been refunded a total of '&6{amount}&a'."
+        economy-claim-buy-cancelled="&cAcquisto annullato! Non puoi acquistare una zona da &6{player}&c. Risultato: &a{result}"
+        economy-claim-buy-confirmation="&6Sei sicuro di voler acquistare questa zona per &a{amount}&6? Clicca per confermare."
+        economy-claim-buy-confirmed="&aHai comprato la zona per &6{amount}&a."
+        economy-claim-buy-not-enough-funds="&cNon hai abbastanza fondi per acquistare questa zona per &a{amount}&c. Attualmente hai un bilancio di &a{balance}&c e ti servono altri &a{amount_required}&c."
+        economy-claim-buy-transfer-cancelled="&cTrasferimento annullato! Impossibile trasferire la proprietà da &6{owner}&c a &6{player}&c. Risultato: &a{result}"
+        economy-claim-not-for-sale="&cQuesta zona non è in vendita."
+        economy-claim-rent-cancel-confirmation="&6Sei sicuro di voler annullare l'affitto&6? Clicca per confermare."
+        economy-claim-rent-cancel-confirmation-balance="&6Sei sicuro di voler annullare l'affitto? Sei in arretrato di &a{balance}&6 che devono essere pagati. Se continui, verrà aggiunto al tuo debito fino a quando non avrai pagato tutto.\nClicca per confermare."
+        economy-claim-rent-cancelled="&aHai annullato il tuo affitto."
+        economy-claim-rent-confirmation="&6Sei sicuro di voler far affittare la tua zona?\n\n&bPrezzo: &a{amount} per &a{type}&6\n\nUna volta che la zona verrà affittata tu perderai l'accesso ad essa.\nClicca per confermare."
+        economy-claim-rent-confirmation-max="&6Sei sicuro di voler far affittare la tua zona?\n\n&bPrezzo: &a{amount} per &a{type}\n&bL'affitto finirà in: &a{max-time}&6\n\nOnce your claim is rented, you will lose access to it.\nClicca per confermare."
```

Comment:
Traduzione incompleta
economy-claim-rent-confirmation-max="&6Sei sicuro di voler far affittare la tua zona?\n\n&bPrezzo: &a{amount} per &a{type}\n&bL'affitto finirà in: &a{max-time}&6\n\nUna volta che il tuo claim sarà affittato, vi perderai accesso.\nClicca per confermare."